content/blog/2021-08-31-aug-community-gems.md

```diff
@@ -0,0 +1,130 @@
+---
+title: August '21 Community Gems
+date: 2021-08-31
+description: |
+  A roundup of technical Q&A's from the DVC community.
+  This month: DVC pipeline configs, working with remotes,
+  file handling and more.
+descriptionLong: |
+  A roundup of technical Q&A's from the DVC community.
+  This month: DVC pipeline configs, working with remotes,
+  file handling and more.
+picture: 2021-08-31/gems-cover.png
+author: milecia_mcgregor
+commentsUrl: https://discuss.dvc.org/t/august-21-community-gems/45342t5t4egrf4w
+tags:
+  - Community
+  - Plots
+  - Pipelines
+  - CML
+  - Git
+---
+
+### [Q: Are TOML files supported for storing model metrics and displaying them via `dvc metrics show`?](https://discord.com/channels/485586884165107732/485596304961962003/865974923079319563)
+
+Thanks for the question @naeljaneLiblikas!
+
+DVC does not support TOML files for metrics. TOML files are only used for
+parameters.
+
+We do have an [open issue](https://github.com/iterative/dvc/issues/6402) for
+this so you can keep up with any updates.
+
+### [Q: How can I use object storage as a data source for MLOps with DVC?](https://discord.com/channels/485586884165107732/485596304961962003/866393535296176148)
```

Comment:
Unfortunately, I don't think this a good Q/A. Question sounds generic enough to provide a very different (simpler) answer - "use DVC remotes". Other answers are pretty advanced and/or optimizations. I would recommend replacing this cc @jorgeorpinel - why did we think this question is about these advanced cases? :)

Reply:
I replaced this Q/A with one about getting CSV files for the experiments table.